bundles/AdminBundle/Resources/public/js/pimcore/document/tree.js

```diff
@@ -284,12 +288,22 @@ pimcore.document.tree = Class.create({
             (node.data.type === 'page' || node.data.type === 'hardlink') &&
             pimcore.globalmanager.get("user").isAllowed('redirects')
         ) {
+            var data = {
+                "id": node.data.id,
+                "params": params,
+                "moveCallback": moveCallback,
+            };
+
+            this.nodesToMove.push(data);
             // ask the user if redirects should be created, if node was moved to a new parent
             Ext.MessageBox.confirm("", t("create_redirects"), function (buttonValue) {
+                let redirectParam = [];
                 if (buttonValue == "yes") {
-                    params['create_redirects'] = 'true';
+                    redirectParam['create_redirects'] = 'true';
```

Comment:
Can't we just use `this.nodesToMove[nodeIdx]['params']['create_redirects'] = 'true';` here?

Reply:
changed it.